sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1473,14 +1510,7 @@ pi_result piEnqueueMemBufferRead(pi_queue Queue, pi_mem Src,
     RetEv->IsDummyEvent = true;
   }
 
-  int Status =
-      buf->CmBufferPtr->ReadSurface(reinterpret_cast<unsigned char *>(Dst),
-                                    nullptr, // event
-                                    static_cast<uint64_t>(Size));
-
-  if (Status != cm_support::CM_SUCCESS) {
-    return PI_INVALID_MEM_OBJECT;
-  }
+  memcpy(Dst, buf->MapHostPtr, Size);
```

Comment:
I would like to discuss design of _pi_buffer on a sync meeting to understand more clearly what happens here and why is the change

Reply:
I'll revert to call `ReadSurface`.